src/components/Footer/Footer.js

```diff
@@ -10,9 +10,9 @@ import MemoIc_arrow_upvote_on from '../../assets/arrow/Ic_upvote_on';
 import MemoIc_block_inactive from '../../assets/block/Ic_block_inactive';
 import MemoIc_comment from '../../assets/icons/Ic_comment';
 import MemoIc_share from '../../assets/icons/Ic_share';
+import {IcDmAnon} from '../../assets/icons/ic_dm_anon';
 import Memoic_globe from '../../assets/icons/ic_globe';
 import MemoIc_senddm from '../../assets/icons/ic_send_dm';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `IcDmAnon` has been moved from a different location and an unused import `SendDMAnonBlock` has been removed. This improves the code by removing unnecessary imports and organizing the remaining ones.
```diff
- import SendDMAnonBlock from '../../assets/icons/images/send-dm-anon-black.svg';
+ import {IcDmAnon} from '../../assets/icons/ic_dm_anon';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->